prov/efa/src/efa_av.c

```diff
@@ -877,13 +832,6 @@ int efa_av_open(struct fid_domain *domain_fid, struct fi_av_attr *attr,
 	av->used = 0;
```

Comment:
if efa_domain_base->type != EFA_DOMAIN_RDM above the EFA_INFO line can be reached with an uninitialized av_attr.flags (better to set to some sentinel value ... at least in DEBUG mode ... so you can tell this is what happened).

Reply:
No sure I understand the comment.